src/main/steps/common/summary/utils.ts

```diff
@@ -88,15 +88,18 @@ export const summaryList = (
   const summaryData: SummaryListRow[] = [];
   for (const key in keys) {
     const keyLabel = keys[key];
+    const getSelectedprivaedet = userCase[key] + getSelectedPrivateDetails(userCase);
+    const setkey = key1 => {
+      if (key1 === 'startAlternative' && userCase[key1] !== 'undefined') {
```

Comment:
is undefined of data type string?

Reply:
This is part of old code itself.I did not made any change on making that as string type.